scanpy/tools/_sim.py

```diff
@@ -410,9 +409,8 @@ def __init__(
         # checks
         if initType not in ['branch', 'random']:
             raise RuntimeError('initType must be either: branch, random')
-        read = False
         if model not in self.availModels.keys():
-            message = 'model not among predefined models \n'
+            message = 'model not among predefined models \n'  # noqa: F841
```

Comment:
this is weird

Reply:
Yeah, it can absolutely be removed I guess :)
I didn't want to do it since I did not know whether you have some magic going on somewhere that looks for these "message" strings to log them or something.
But I guess we should remove or replace this with a log statement/exception here